test/stripe/tokens_test.exs

```diff
@@ -77,7 +77,7 @@ defmodule Stripe.TokensTest do
       {:ok, res} ->
         #Apex.ap res
         assert res.id
-        assert res.status == "paid"
+        assert res.status == "succeeded"
```

Comment:
I had to change this to make `tokens_test.exs` pass, using the latest API according to my strip dashboard.